### ISSUE TITLE
pandas from_items deprecation warning

### ISSUE THREAD

**Issue by summerela** [cggh/scikit-allel#226]
I receive the following warning every time I read in a VCF file with scikit-allel: 

```
/usr/lib64/python3.6/site-packages/allel/io/vcf_read.py:1569: FutureWarning: from_items is deprecated. Please use DataFrame.from_dict(dict(items), ...) instead. DataFrame.from_dict(OrderedDict(items)) may be used to preserve the key order.
  df = pandas.DataFrame.from_items(items)
```

**Comment by alimanfoo** [comment]
Thanks @summerela. Looks like it should be a straightforward fix, ``from_items()`` is used in [one place](https://github.com/cggh/scikit-allel/blob/master/allel/io/vcf_read.py#L1774), and the warning suggests we could just replace ``from_items(items)`` with ``from_dict(OrderedDict(items))``. Pull request welcome.

**Comment by alimanfoo** [comment]
Resolved via #228
